### PR TITLE
Add Metakgp branding to MFTP Mails

### PR DESCRIPTION
**Pull request by thealphadollar**
As discussed in the metakgp slack channel, we should brand the mails we sent through MFTP in order to reach more people and make them aware of us. This way we can also hope to get people involved in the process of contributing.

### CODE REVIEW

**Comment by amrav** [comment]
Can you post a screenshot of what this looks like on an actual notice?

Personally I would opt for more subtle branding, given the importance and frequency of these emails. Eg smaller font, greyed out, like a footer, possibly separated from the main body by a <hr>.

As for the link, maybe we can put a more direct call to action. Linking to the github page doesn't give people an easy to understand overview. metakgp.github.io is probably a better landing page.

**Comment by thealphadollar** [comment]
@amrav We already have the MFTP google group notice at the bottom and hence this doesn't make much difference. There is a separation of 4 `<br/>` tags which come with the notice itself from the scraper.

**Comment by thealphadollar** [comment]
The link has been updated.

I believe the current text is minimal enough; adding a logo would be worse. I'm saying this since we already get the default Google Group notice at the end and a textual line goes well with it rather than a logo and then the text.

File: hooks.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 if 'NOTICES_EMAIL_ADDRESS' not in env:
     env['NOTICES_EMAIL_ADDRESS'] = env['EMAIL_ADDRESS']
+METAKGP_BRANDING = "Brought To Your Inbox By <a href='https://metakgp.github.io'>Metakgp</a> With <3"
```
Comment (icyflame) [inline hooks.py]: No need for `Title Case` here. This is better: `Brought to you by Metakgp`

**Comment by thealphadollar** [comment]
Please have a look now. @amrav @icyflame

**Pull request merged**